lib/ts/recipe/session/claimBaseClasses/primitiveClaim.ts

```diff
@@ -0,0 +1,80 @@
+import { Awaitable, JSONValue } from "../../../types";
+import { SessionClaimBuilder, SessionClaimValidator } from "../types";
+
+export abstract class PrimitiveClaim<T extends JSONValue> extends SessionClaimBuilder<T> {
+    constructor(key: string) {
+        super(key);
+    }
+
+    abstract fetch(userId: string, userContext: any): Awaitable<T | undefined>;
```

Comment:
We should consider renaming this function to something like `fetchValue` - calling it just `fetch` was confusing for others in the team

Reply:
I think `fetchValue` is good.